training/distributed_training/pytorch/data_parallel/roberta/roberta.ipynb

```diff
@@ -0,0 +1,412 @@
+   "id": "a36893f2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Configure hyper-parameters\n",
+    "# RoBERTa Large 1.3B parameters\n",
+    "hyperparameters = {\n",
+    "    'fp16': '',\n",
+    "    'task': 'masked_lm',\n",
+    "    'criterion': 'masked_lm',\n",
+    "    'arch': 'roberta_large',\n",
+    "    'sample-break-mode': 'complete',\n",
+    "    'tokens-per-sample': 512,\n",
+    "    'optimizer': 'adam',\n",
+    "    'adam-eps': 1e-6,\n",
+    "    'clip-norm': 0.0,\n",
+    "    'lr-scheduler': 'polynomial_decay',\n",
+    "    'lr': 0.0001,\n",
+    "    'warmup-updates': 10000,\n",
+    "    'total-num-update': 125000,\n",
+    "    'dropout': 0.1,\n",
+    "    'attention-dropout': 0.1,\n",
+    "    'weight-decay': 0.01,\n",
+    "    'max-sentences': 8,\n",
+    "    'update-freq': 1,\n",
+    "    'max-update': 125000,\n",
+    "    'log-format': 'simple',\n",
+    "    'log-interval': 10,\n",
+    "    'encoder-layers': 24,\n",
+    "    'encoder-embed-dim': 2048,\n",
+    "    'encoder-ffn-embed-dim': 8192,\n",
+    "    'memory-efficient-fp16': '',\n",
+    "    'distributed-no-spawn' : '',\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "58027f6d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Configure hyper-parameters\n",
+    "# RoBERTa Large 350M parameters\n",
+    "hyperparameters = {\n",
+    "    'fp16': '',\n",
+    "    'task': 'masked_lm',\n",
+    "    'criterion': 'masked_lm',\n",
+    "    'arch': 'roberta_large',\n",
+    "    'sample-break-mode': 'complete',\n",
+    "    'tokens-per-sample': 512,\n",
+    "    'optimizer': 'adam',\n",
+    "    'adam-eps': 1e-6,\n",
+    "    'clip-norm': 0.0,\n",
+    "    'lr-scheduler': 'polynomial_decay',\n",
+    "    'lr': 0.0001,\n",
+    "    'warmup-updates': 10000,\n",
+    "    'total-num-update': 125000,\n",
+    "    'dropout': 0.1,\n",
+    "    'attention-dropout': 0.1,\n",
+    "    'weight-decay': 0.01,\n",
+    "    'max-sentences': 16,\n",
+    "    'update-freq': 1,\n",
+    "    'max-update': 125000,\n",
+    "    'log-format': 'simple',\n",
+    "    'log-interval': 10,\n",
+    "    'distributed-no-spawn' : ''\n",
+    "}\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "06362229",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job_name = \"pytorch-smdataparallel-roberta-testrun-new-16\"  # This job name is used as prefix to the sagemaker training job. Makes it easy to look for your training job in SageMaker Training job console.\n",
+    "estimator = PyTorch(\n",
+    "    entry_point=\"entry_point.py\",\n",
+    "    role=role,\n",
+    "    image_uri=docker_image,\n",
+    "    source_dir=\".\",\n",
+    "    instance_count=instance_count,\n",
+    "    instance_type=instance_type,\n",
+    "    framework_version=\"1.8.1\", # this value does not matter since we provided our own image\n",
```

Comment:
to avoid confusion, can this be removed. Or at-least set to actual version v1.9.0?